components/composition/Wrap/Wrap.css

```diff
@@ -0,0 +1,67 @@
+/* stylelint-disable selector-no-qualifying-type */
```

Comment:
We should set this rule to ignore attribute https://stylelint.io/user-guide/rules/selector-no-qualifying-type/#ignore-attribute-class-id